components/brave_today/browser/feed_parsing.cc

```diff
@@ -61,6 +63,10 @@ bool ParseFeedItem(const base::Value& feed_item_raw,
     VLOG(1) << "Could not parse item url: " << url_raw;
     return false;
   }
+  if (!url.SchemeIsHTTPOrHTTPS()) {
+    VLOG(1) << "Item url was not HTTP or HTTPS: " << url.spec();
+    return false;
+  }
```

Comment:
This bypasses non-http(s) items from the combined feed hosted by Brave

Reply:
Is there a corresponding backend change? I think ideally we wouldn't be including any non-https items in the combined feed in the first place.

Reply:
I would hope it wouldn't include it but doesn't hurt to double-check. And there's always the possibility someone points the browser to a third party combined feed source via the cli param...